test/controllers/spam_controller_test.rb

```diff
@@ -10,15 +10,20 @@ def setup #called before every single test
 
   def maps_custom_setup(spam_maps)
     if spam_maps
-      @maps.collect { |map|
+      @maps.uniq.collect { |map|
         map.spam
         map.user.ban
       }
     end
     @map_ids = @maps.collect(&:id).join(',')
   end
 
-  def users_custom_setup
+  def users_custom_setup(ban_users)
```

Comment:
Here, using a boolean as the attribute can make it a little unclear how it's being used lower down. I know this is just a test, but could we as a good habit try:

```suggestion
  def users_custom_setup(ban_users: true) # setting default true?
```

then the usage would be:

```ruby
users_custom_setup(ban_users: false)
```

It's a little more verbose but much clearer where you're using the method lower down. What do you think?

Reply:
I think this makes perfect sense @jywarren, thank you. I've implemented the changes. It does look better now 😄